tembo-operator/src/network_policies.rs

```diff
@@ -290,25 +290,23 @@ pub async fn reconcile_network_policies(client: Client, namespace: &str) -> Resu
             "policyTypes": ["Egress"],
             "egress": [
                 {
+                    "ports": [
+                        {
+                            "port": 8080,
+                            "protocol": "TCP"
+                        }
+                    ],
                     "to": [
                         {
-                            "namespaceSelector": {
-                                "matchLabels": {
-                                    "kubernetes.io/metadata.name": "tembo-ai"
-                                }
-                            },
-                            "podSelector": {
-                                "matchLabels": {
-                                    "app.kubernetes.io/name": "tembo-ai-gateway"
-                                }
```

Comment:
Does removing this block break the path to connect via cluster IP within the same cluster to the tembo-ai services?

Reply:
It does. I can add it back if needed to allow for both